cmd/sync.go

```diff
@@ -329,6 +334,8 @@ type cookedSyncCmdArgs struct {
 	excludePaths          []string
 	includeFileAttributes []string
 	excludeFileAttributes []string
+	includeRegex		  []string
+	excludeRegex		  []string
```

Comment:
minor: double check if spacing is consistent

Reply:
`go fmt` may help.